src/components/Hero/HubHero/index.tsx

```diff
@@ -1,9 +1,10 @@
-import { Box, Heading, Stack, Text } from "@chakra-ui/react"
-
 import type { CommonHeroProps } from "@/lib/types"
 
 import { CallToAction } from "@/components/Hero/CallToAction"
```

Comment:
Can we also include the migration of this component?

Reply:
I think it's doable